src/ContextMenu/ContextMenu.Item.tsx

```diff
@@ -24,43 +25,58 @@ export interface IContextMenuItem<TValue = any> {
   iconColor?: keyof OperationalStyleConstants["color"]
   onClick?: ContextMenuProps["onClick"]
   value?: TValue
+  isActive?: boolean
 }
 
-const Container = styled("div")<Props>(({ align, theme, onClick, condensed, width, item }) => ({
-  userSelect: "none",
-  label: "contextmenuitem",
-  width: width || (condensed ? 160 : "100%"),
-  whiteSpace: "nowrap",
-  overflow: "hidden",
-  textOverflow: "ellipsis",
-  backgroundColor: theme.color.white,
-  lineHeight: `${condensed ? 35 : 44}px`,
-  padding: `0 ${theme.space.content}px`,
-  textAlign: align,
-  display: "flex",
-  alignItems: "center",
-  ...(Boolean(typeof item !== "string" && item.description)
-    ? {
-        borderBottom: `1px solid ${theme.color.separators.default}`,
-      }
-    : {}),
-  ...(!!onClick
-    ? {
-        cursor: "pointer",
-        color: theme.color.text.default,
-        "&:hover": {
-          backgroundColor: darken(theme.color.white, 2),
-        },
-      }
-    : {
-        cursor: "not-allowed",
-        color: theme.color.text.lightest,
-      }),
-  borderTop: `1px solid ${theme.color.separators.default}`,
-  "&:last-child": {
-    paddingBottom: 2,
-  },
-}))
+const Container = styled("div")<Props>(({ align, theme, onClick, isActive, condensed, width, item }) => {
```

Comment:
```suggestion
const ContextMenuItem = styled("div")<Props>(({ align, theme, onClick, isActive, condensed, width, item }) => {
```

based on `label: "contextmenuitem",`?

Reply:
This constant is declared further down as the actual component. This is just a styled visual container.